docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,194 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
+This procedure describes how to display Sawtooth network statistics and
+operating system metrics in Grafana, using Docker, InfluxDB, and Telegraf.
+
+Requirements
+============
+
+This guide assumes Docker and one or more sawtooth validators have been
+configured.
+
+Download sawtooth-core from the Hyperledger GitHub repository.
```

Comment:
I suggest adding the link to the repo, either as a raw link or by turning "sawtooth-core" into a link.